### PR TITLE
Updates to RN 0.25+

### PR DESCRIPTION
**Pull request by samin**
React API separated from React Native
https://github.com/facebook/react-native/releases/tag/v0.25.1

### CODE REVIEW

File: SmartScrollView.js [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import React, {
   Component,
+  PropTypes,
+} from 'react';
+
+import {
```
Comment (mlabrum) [inline SmartScrollView.js]: Could you also
`import ReactNative, {`

and update https://github.com/samin/react-native-smart-scroll-view/blob/eb44f6ddb63bb8c6fa5851aaab6d9b50041919f1/SmartScrollView.js#L123 from `React.findNodeHandle(this._smartScroll);` to `ReactNative.findNodeHandle(this._smartScroll);`

**Comment by nikhilaravi** [comment]
@samin do you want to make the update suggested by @mlabrum ?

**Comment by samin** [comment]
@nikhilaravi @mlabrum done

**Comment by nikhilaravi** [comment]
@samin great! do you want to change the package.json version to 1.3.2?

**Comment by nikhilaravi** [comment]
thanks @samin

**Pull request merged**